src/dialog.jsx

```diff
@@ -410,61 +424,113 @@ const DialogInline = React.createClass({
 
 });
 
-
 const Dialog = React.createClass({
 
   propTypes: {
+    /**
+     * **DEPRECATED** The `ref` of the action to focus on when the `Dialog` is displayed.
```

Comment:
What about using this https://github.com/react-bootstrap/react-prop-types/blob/master/src/deprecated.js?